### PR TITLE
Add community values, improve code of conduct

### PR DESCRIPTION
**Pull request by jpkrohling**
This PR improves our code of conduct by stating how violations should be reported. It also adds a set of community values, showing what we expect from our community members when interacting with other members.

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: code-of-conduct.md
Comment (yurishkuro) [inline code-of-conduct.md:35]:
```suggestion
cases, the community member might be requested to depart from our communities,
```

File: mission-vision-values.md
Comment (bhs) [inline mission-vision-values.md:146]: do we want to explicitly mention conflicts regarding employers (and/or customers of employers)? the "hats" below are all OTel-specific hats...

File: mission-vision-values.md
Comment (bhs) [inline mission-vision-values.md:136]: mega-nitpick, sorry: in the sections above, we used `###` (three levels) for this next level of headers, but here we use `####` (four levels). we should be consistent...

**Review by bhs — COMMENTED** [review]
two minor questions...

@jpkrohling I see that the prev copy I responded to on zoom is gone now... that's evolved into "respectfully disagree" ? Fine with me, just confirming.

File: mission-vision-values.md
Comment (jpkrohling) [reply mission-vision-values.md:136]: I like mega-nitpick, keep them coming.

File: mission-vision-values.md
Comment (jpkrohling) [reply mission-vision-values.md:146]: Can you give me an example of such conflict? I think I see what you mean, but when taking the previous point into consideration ("wear the project's hat"), I'm not sure what else to cover.

I drafted something like this, but wasn't 100% happy:

> a Collector maintainer might be part of the Governance Committee, a JavaScript maintainer might be part of the Technical Committee, a Go approver might have a customer being affected by a specific behavior on the Go's SDK, ...

**Comment by jpkrohling** [comment]
>  I see that the prev copy I responded to on zoom is gone now... that's evolved into "respectfully disagree" ? Fine with me, just confirming.

I think I shared a gist on Zoom, which should still be available, but I do remember a written comment from you somewhere, I just can't find it now :-/ The version here is pretty much the same as the gist, split into the different documents as discussed during the GC call.

**Review by trask — COMMENTED** [review]
~(I saw you asked for more mega-nitpicking 😅)~

~`s/’/'/g`~
~`s/[“”]/"/g`~

nevermind TIL looks like we use both kinds of quotes quite (in)consistently in this repo

**Review by chalin — COMMENTED** [review]
For copyedits, and a link URL change suggestion, see inline comments. /cc @svrnm

**Review by mtwo — APPROVED** [review]

File: mission-vision-values.md
Comment (jpkrohling) [reply mission-vision-values.md]: I'm using unacceptable for now to unblock this, but I think another adjective might be warranted.

File: mission-vision-values.md
Comment (jpkrohling) [reply mission-vision-values.md]: I'm marking this as resolved, as we can tweak this later.

File: code-of-conduct.md
Comment (jpkrohling) [reply code-of-conduct.md:20]: I'm marking this as resolved.  The only 👍🏽 missing is @alolita's, but I do think she expressed her consent in other channels (GC Slack channel?)

File: code-of-conduct.md
Comment (dyladan) [inline code-of-conduct.md:11]: "covered by that in more serious violations" reads weirdly to me.

```suggestion
This project is bound by and abides by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
Conflicts should be resolved respectfully between the involved parties, however
we understand that it may not always be possible without outside help or mediation.
When needed, any community member can ask for help or report a Code of Conduct
violation to the Governance Committee as outlined below.
The project prefers that violations to be reported first to the Governance Committee
so that they may be resolved internally, however, in extreme cases code of conduct
violations should be reported to the CNCF as outlined in the
[CNCF Incident Resolution Procedures](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md).
Anyone who is unsure if a violation should be reported to the project or the CNCF should
refer to the [CNCF CoC Jurisdiction Policy](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-committee-jurisdiction-policy.md).
```

File: code-of-conduct.md
Comment (dyladan) [inline code-of-conduct.md:20]: I think it may be a good idea to provide a more specific example and be more direct about this. I'd take a look at the CNCF's CoC reporting procedure for inspiration https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md#submit-in-writing

We may want to just take their wording almost exactly substituting our GC for their CoC Committee

> If you do not want your report to be received by all members of the CoC Committee, either because you want to submit a report anonymously or because one of the CoC Committee members has a conflict of interest, you may send your report directly to any individual member

File: code-of-conduct.md
Comment (dyladan) [inline code-of-conduct.md:21]: I don't think all reports should always be brought to the entire group and I wouldn't promise that here. It may prevent someone from raising a complaint against a GC member.

Comment (jpkrohling) [reply code-of-conduct.md:21]: Agreed, I replaced "will" by "may":

> any one of us can receive your report and may bring it to the entire group for investigation

File: code-of-conduct.md
Comment (jpkrohling) [reply code-of-conduct.md]: That doc is great. I reorganized the final part of our code of conduct with parts from that. Thank you for the hint! Can you please take another look?

**Review by tedsuo — APPROVED** [review]

**Review by bhs — APPROVED** [review]

File: mission-vision-values.md
Comment (trask) [inline mission-vision-values.md:174]:
```suggestion
tempting to make an ad hominem attack but it’s always wrong. If you have reasons
```

**Review by trask — APPROVED** [review]

File: mission-vision-values.md
Comment (reyang) [inline mission-vision-values.md:186]: Minor, since the link hinted a particular country which might be considered as bias, should we avoid it here?

**Review by reyang — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

File: code-of-conduct.md
Comment (tigrannajaryan) [inline code-of-conduct.md:54]: Nit: I am not sure we need to refer anonymous reporting in this paragraph. Contacting an individual GC member can definitely be more desirable in some cases, so perhaps we just specify the mechanics of how to do it without necessarily trying it to the idea of anonymity. Especially since this paragraph does not say how the contacting member should specify in their request that they want to remain anonymous.

File: mission-vision-values.md
Comment (tigrannajaryan) [inline mission-vision-values.md:125]: This seems like a copy of what is in code-of-conduct.md. Any reason we need this in 2 places?

**Review by tigrannajaryan — APPROVED** [review]
LGTM, just minor comments to improve, but can merged and improved later.

File: code-of-conduct.md
Comment (arminru) [inline code-of-conduct.md:9]:
```suggestion
Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```

File: code-of-conduct.md
Comment (arminru) [inline code-of-conduct.md:28]:
```suggestion
When the Governance Committee (GC) determines that a code of conduct violation
```

File: mission-vision-values.md
Comment (arminru) [inline mission-vision-values.md:175]:
```suggestion
to believe the person you are debating with is not acting in the interest of the
```

**Review by arminru — APPROVED** [review]

File: mission-vision-values.md
Comment (jpkrohling) [reply mission-vision-values.md:186]: Not sure I see it the same way you do, but given @arminru seems to agree with you, I'll reword this.

File: mission-vision-values.md
Comment (jpkrohling) [reply mission-vision-values.md:125]: I think it's a good introduction to what comes next, serving both purposes well (mission/vision/values and CoC). As I see no harm in having it at both places, I'll leave it there for now.

File: code-of-conduct.md
Comment (jpkrohling) [reply code-of-conduct.md:54]: During a GC call, we thought the following doc would be a good source for us to adapt, and it's where this section comes from: https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md#how-to-submit-a-report 

Personally, I like the option of having this explicitly listed here, even though most of it is already mentioned elsewhere (how to find GC members, how to report, ...)

> Especially since this paragraph does not say how the contacting member should specify in their request that they want to remain anonymous.

Not sure I understood this part. "Let them know you would like to report anonymously" should suffice here, I believe. Or do you mean that people might want to remain completely anonymous even to the party they are reporting the issue to?

**Comment by jpkrohling** [comment]
Thank you all for your comments! I'm merging this but will follow-up with @tigrannajaryan to work on the remaining item. Once that is clarified, I'll update the .github's version of the CoC, so that it's propagated to all repositories.

File: code-of-conduct.md
Comment (jpkrohling) [reply code-of-conduct.md:54]: Tracked here: https://github.com/open-telemetry/community/issues/1684

File: mission-vision-values.md
Comment (alolita) [inline mission-vision-values.md:144]: Suggestion: Change sentence to --> Each member’s priorities when acting as a maintainer or contributor for the project should represent the interests of the project and its community first before their other obligations (e.g. employer).

File: mission-vision-values.md
Comment (alolita) [inline mission-vision-values.md:149]: Change "might" to "may".

**Review by alolita — COMMENTED** [review]
@jpkrohling @dyladan @trask - providing my suggestions. I see the PR has been merged but would like to see if these suggestions can be incorporated for further clarity. Thanks!

**Comment by trask** [comment]
> @jpkrohling @dyladan @trask - providing my suggestions. I see the PR has been merged but would like to see if these suggestions can be incorporated for further clarity. Thanks!

I sent #1855 to incorporate your comments

**Pull request merged**
